code/modules/mob/living/carbon/human/species.dm

```diff
@@ -1080,7 +1080,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 				return FALSE
 			return TRUE
 		if(ITEM_SLOT_SUITSTORE)
-			if(HAS_TRAIT(I, TRAIT_NODROP))
+			if(HAS_TRAIT_NOT_FROM(I, TRAIT_NODROP, CHANGELING_TRAIT))
```

Comment:
This is pretty awful. The trait source shouldn't be modifying the functionality.

Reply:
I agree, but I did not figure out a better way, as all flesh disguise items are NODROP.

Reply:
What exactly is the issue this is fixing.

Reply:
Flesh disguise is supposed to also copy the item in the suit store slot, but it never does due to this check, as they are set to NODROP, so they always fail here and are deleted afterwards.

Reply:
You could apply the NODROP after equipping i guess. But that's still pretty ugly. Not sure exactly what scenario is current code trying to prevent. Dropping undroppable items in pocket by dropping the suit ? Pretty roundabout way to do this.

Reply:
As you suggested, NODROP is now only applied after actually equipping the flesh item via transformation.